packages/broker/src/plugins/operator/OperatorPlugin.ts

```diff
@@ -27,6 +28,8 @@ export interface OperatorServiceConfig {
     signer: Signer
     operatorContractAddress: EthereumAddress
     theGraphUrl: string
+    maxSponsorshipsCount?: number
```

Comment:
Maybe a more specific name if this limits just the operator value maintenance, not anything else?

Reply:
maybe maxSponsorshipsInWithdraw

Reply:
Updated